pom.xml

```diff
@@ -428,6 +433,10 @@
         <carbon.kernel.carbon.base.pkg.version>[1.0.0, 2.0.0)</carbon.kernel.carbon.base.pkg.version>
         <axis2.osgi.version.range>[1.6.1, 2.0.0)</axis2.osgi.version.range>
 
+        <!--Carbon Database Utils Version-->
+        <org.wso2.carbon.database.utils.version>2.1.0</org.wso2.carbon.database.utils.version>
+        <org.wso2.carbon.database.utils.version.range>[2.0.0,2.2.0)</org.wso2.carbon.database.utils.version.range>
```

Comment:
If we want to the 2.1.0 version, shall we restrict the range to start with 2.1.0?

Reply:
Fixed with: https://github.com/wso2-extensions/identity-event-handler-notification/pull/230/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R444